config/requirements.txt

```diff
@@ -1,51 +1,100 @@
-backports.ssl-match-hostname==3.5.0.1
+ansible-pylibssh>=0.2.0 # ansible/netcommon, cisco/asa, cisco/ios, cisco/iosxr
+apache-libcloud # legacy
+awxkit # awx/awx
+azure-cli-core==2.34.0 # azure/azcollection
+azure-common==1.1.11 # azure/azcollection
+azure-containerregistry==1.1.0 # azure/azcollection
+azure-graphrbac==0.61.1 # azure/azcollection
+azure-identity==1.7.0 # azure/azcollection
+azure-keyvault==1.1.0 # azure/azcollection
+azure-mgmt-apimanagement==3.0.0 # azure/azcollection
+azure-mgmt-authorization==2.0.0 # azure/azcollection
+azure-mgmt-automation==1.0.0 # azure/azcollection
+azure-mgmt-batch==5.0.1 # azure/azcollection
+azure-mgmt-cdn==11.0.0 # azure/azcollection
+azure-mgmt-compute==26.1.0 # azure/azcollection
+azure-mgmt-containerinstance==9.0.0 # azure/azcollection
+azure-mgmt-containerregistry==9.1.0 # azure/azcollection
+azure-mgmt-containerservice==20.0.0 # azure/azcollection
+azure-mgmt-core==1.3.0 # azure/azcollection
+azure-mgmt-cosmosdb==6.4.0 # azure/azcollection
+azure-mgmt-datafactory==2.0.0 # azure/azcollection
+azure-mgmt-datalake-store==1.0.0 # azure/azcollection
+azure-mgmt-devtestlabs==9.0.0 # azure/azcollection
+azure-mgmt-dns==8.0.0 # azure/azcollection
+azure-mgmt-eventhub==10.1.0 # azure/azcollection
+azure-mgmt-hdinsight==9.0.0 # azure/azcollection
+azure-mgmt-iothub==2.2.0 # azure/azcollection
+azure-mgmt-keyvault==10.0.0 # azure/azcollection
+azure-mgmt-loganalytics==12.0.0 # azure/azcollection
+azure-mgmt-managedservices==6.0.0 # azure/azcollection
+azure-mgmt-managementgroups==1.0.0 # azure/azcollection
+azure-mgmt-marketplaceordering==1.1.0 # azure/azcollection
+azure-mgmt-monitor==3.0.0 # azure/azcollection
+azure-mgmt-network==19.1.0 # azure/azcollection
+azure-mgmt-notificationhubs==7.0.0 # azure/azcollection
+azure-mgmt-nspkg==2.0.0 # azure/azcollection
+azure-mgmt-privatedns==1.0.0 # azure/azcollection
+azure-mgmt-rdbms==10.0.0 # azure/azcollection
+azure-mgmt-recoveryservices==2.0.0 # azure/azcollection
+azure-mgmt-recoveryservicesbackup==3.0.0 # azure/azcollection
+azure-mgmt-redis==13.0.0 # azure/azcollection
+azure-mgmt-resource==21.1.0 # azure/azcollection
+azure-mgmt-search==8.0.0 # azure/azcollection
+azure-mgmt-servicebus==7.1.0 # azure/azcollection
+azure-mgmt-sql==3.0.1 # azure/azcollection
+azure-mgmt-storage==19.0.0 # azure/azcollection
+azure-mgmt-trafficmanager==1.0.0b1 # azure/azcollection
+azure-mgmt-web==6.1.0 # azure/azcollection
+azure-nspkg==2.0.0 # azure/azcollection
+azure-storage-blob==12.11.0 # azure/azcollection
 boto3>=1.18.0 # amazon/aws, community/aws
 botocore>=1.21.0 # amazon/aws, community/aws
-deprecation>=2.0
-google-auth==1.6.2
-ipaddress==1.0.23
-monotonic==1.4
-ncclient>=0.6.3
-netaddr==0.7.19
-openstacksdk==0.23.0
-ovirt-engine-sdk-python==4.2.4
-pexpect==4.6.0
-psutil==5.6.6 # Match the version installed directly into the venv
-pykerberos==1.2.1
+ciscoisesdk>=2.0.8 # cisco/ise
+cryptography>=36.0.0 # cisco/intersight, cisco/aci, microsoft/ad
```

Comment:
This matches rpm as well which is `python3-cryptography-36.0.1-4.el9.x86_64`, so this is good.